### PR TITLE
Fix/README file for GA

### PR DESCRIPTION
**Pull request by shay-berman**
1. Update support matrix
2. Add ibm knowlage center link
3. Remove install\uninstall section, instead add link to the operator github repo for all the detail of the installation. 
4. update DISCLAIMER

---
This change is [Reviewable](https://reviewable.io/reviews/ibm/ibm-block-csi-driver/103)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Supported operating systems:
 
 DISCLAIMER: The driver is provided as is, without warranty.
 
-Full documentation can be found in the IBM knowlage center -> HERE
+Full documentation can be found in the [IBM knowlage center](https://www.ibm.com/support/knowledgecenter/SSRQ8T).
```
Comment (barakda) [inline README.md]: The link is broken, i believe it is because we haven't created the page yet, right?

Comment (rivkap) [reply README.md]: correct. it will only go live at GA.

File: README.md [hunk 2]
```diff
@@ -62,52 +64,36 @@ multipath -ll
 **Important:** When configuring Linux multipath devices, verify that the `find_multipaths` parameter in the `multipath.conf` file is disabled. In RHEL 7.x, remove the`find_multipaths yes` string from the `multipath.conf` file.
 
 #### 3. Configure storage system connectivity
-3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN or IQN of the node. 
+3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN(for Fibre Channel) or IQN(for iSCSI) of the node.
```
Comment (rivkap) [inline README.md]: add spaces before each bracket "("

File: README.md [hunk 3]
```diff
@@ -62,52 +64,36 @@ multipath -ll
...
 
 3.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
 
 3.3. For iSCSI, perform the following steps:
 
 3.3.1. Make sure that the login to the iSCSI targets is permanent and remains available after a reboot of the worker node. To do this, verify that the node.startup in the /etc/iscsi/iscsid.conf file is set to automatic. If not, set it as required and then restart the iscsid service `$> service iscsid restart`.
 
-3.3.2. Discover and log into at least two iSCSI targets on the relevant storage
-systems.
+3.3.2. Discover and log into at least two iSCSI targets on the relevant storage systems. (NOTE: Without at least two ports, multipath device will not be created.)
```
Comment (rivkap) [inline README.md]: (NOTE: Without at least two ports, **a** multipath device will not be created.)

File: README.md [hunk 4]
```diff
@@ -120,37 +106,10 @@ OR
 ## Installing the driver
 This section describes how to install the CSI driver.
 
-```sh
-###### Download the driver yml file from github:
-$> curl https://raw.githubusercontent.com/IBM/ibm-block-csi-driver/master/deploy/kubernetes/v1.13/ibm-block-csi-driver.yaml > ibm-block-csi-driver.yaml 
+From this version(1.0.0) the deployment method of the driver is done via `Operator for IBM Block CSI Driver` -> https://github.com/ibm/ibm-block-csi-operator.
 
-###### Optional: Only edit the `ibm-block-csi-driver.yaml` file if you need to change the driver IMAGE URL. By default, the URL is `ibmcom/ibm-block-csi-driver-controller:1.0.0` and `ibmcom/ibm-block-csi-driver-node:0.9.0`.
+Note: This `deploy/kubernetes` yaml files are deprecated from version v1.0.0.
```
Comment (rivkap) [inline README.md]: NOTE: The `deploy/kubernetes` yaml file is deprecated from version v1.0.0.

File: README.md [hunk 5]
```diff
@@ -315,21 +273,8 @@ No resources found.
...
-```sh
-$> kubectl delete storageclass/gold
-$> kubectl delete -n kube-system secret/a9000-array1
-$> kubectl delete -f ibm-block-csi-driver.yaml
-
-##### Kubernetes version 1.13 automatically creates the CSIDriver `block.csi.ibm.com`, but it does not delete it automatically when removing the driver manifest. In order to clean up the CSIDriver object, run the following command:
-$> kubectl delete CSIDriver block.csi.ibm.com
-
-```
+From this version(1.0.0) the deployment method of the driver is done via `Operator for IBM Block CSI Driver` -> https://github.com/ibm/ibm-block-csi-operator.
```
Comment (rivkap) [inline README.md]: add a space before the bracket "("

File: README.md [hunk 6]
```diff
@@ -15,7 +15,9 @@ Supported IBM storage systems:
 Supported operating systems:
   - RHEL 7.x (x86 architecture)
 
-DISCLAIMER: The driver is provided as is, without warranty. Version 1.0.0 of the IBM block storage CSI driver is a beta software version. Do not use this driver for production purposes and do not contact IBM for support. Any issue will be handled on a best-effort basis. 
+DISCLAIMER: The driver is provided as is, without warranty.
+
+Full documentation can be found in the [IBM knowledge center](www.ibm.com/support/knowledgecenter/SSRQ8T).
```
Comment (rivkap) [inline README.md]: found **on** IBM knowledge center

**Pull request merged**